src/com/goide/GoAnnotator.java

```diff
@@ -133,6 +137,12 @@ private static TextAttributesKey getColor(GoFieldDefinition o) {
               : GoSyntaxHighlightingColors.STRUCT_LOCAL_MEMBER;
   }
 
+  private static TextAttributesKey getColor(GoFieldNameImpl o) {
```

Comment:
Also, the implementation has access to the ```getName()``` method which the interface doesn't expose.